packages/store-sync/src/postgres/buildInternalTables.ts

```diff
@@ -0,0 +1,32 @@
+import { integer, pgSchema, text } from "drizzle-orm/pg-core";
+import { DynamicAbiType, StaticAbiType } from "@latticexyz/schema-type";
+import { transformSchemaName } from "./transformSchemaName";
+import { asAddress, asBigInt, asJson, asNumber } from "./columnTypes";
+
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+export function buildInternalTables() {
+  const schema = pgSchema(transformSchemaName("__mud_internal"));
+  return {
+    chain: schema.table("chain", {
+      // TODO: change schema version to varchar/text?
+      schemaVersion: integer("schema_version").notNull().primaryKey(),
```

Comment:
is this already in preparation for a protocol type?

Reply:
it's a reflection of the `schemaVersion.ts` file (we have this in the sqlite indexer already) to help detect when the indexer meta tables change (the shape of the indexer DB as opposed to the shape of the MUD tables) so we can recreate them

I opened an issue for a better approach: https://github.com/latticexyz/mud/issues/1362